app/master/build.py

```diff
@@ -11,17 +11,17 @@
 from typing import Dict
 
 from app.common.build_artifact import BuildArtifact
+from app.common.metrics import build_state_duration_seconds, serialized_build_time_seconds, internal_errors, ErrorType
 from app.master.build_fsm import BuildFsm, BuildEvent, BuildState
 from app.master.build_request import BuildRequest
 from app.project_type.project_type import ProjectType
 from app.util import util
 from app.util.conf.configuration import Configuration
 from app.util.counter import Counter
 from app.util.exceptions import ItemNotFoundError
-import app.util.fs
 from app.util.log import get_logger
 from app.util.single_use_coin import SingleUseCoin
-from app.common.metrics import build_state_duration_seconds, serialized_build_time_seconds
+import app.util.fs
```

Comment:
I sorted this with vim by piping the imports through the sort bash command. The sort bash command put it at the end because it starts with `import` and all the other lines start with `from`. I put it back (intellij wins this one)